ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/dropdown/dropdown.tsx

```diff
@@ -100,10 +101,11 @@ export type withContext = {
   dropdownContext: Context
 }
 
-type Props = {
+export type Props = {
```

Comment:
Should we rename to `DropdownProps` now that it's being exported?

Reply:
Importers can rename it when they import it.  I might remove this though, I don't know if I ended up needing it to actually be exported.